src/VecSim/algorithms/hnsw/hnsw_tiered.h

```diff
@@ -126,6 +126,9 @@ class TieredHNSWIndex : public VecSimTieredIndex<DataType, DistType> {
     // Returns the id that the job was stored under (to be set in the job id field).
     idType setAndSaveInvalidJob(AsyncJob *job);
 
+    // Handle deletion of vector inplace condering that async deletion might occurred beforhand.
```

Comment:
?
```suggestion
    // Handle deletion of vector inplace considering that async deletion might occurred beforehand.
```

Reply:
Sorry, I sat on my keyboard I guess ;)
The suggestion is right, is there anything else unclear?

Reply:
just wasn't sure you meant "considering"